pyalgostrategypool/macd_crossover_v2.py

```diff
@@ -0,0 +1,67 @@
+import talib
```

Comment:
since we are removing `macd_crossover.py` from this PR anyways,

copy this code into that file, keep that file, and delete this one

change the name also inside the code accordingly

Reply:
Done